packages/react-components/react-dialog/src/components/Dialog/Dialog.cy.tsx

```diff
@@ -179,6 +179,7 @@ describe('Dialog', () => {
         }
       }, [open]);
       return (
+        //eslint-disable-next-line react/jsx-no-bind
```

Comment:
@TristanWatanabe I don't expect this to be triggered for test files, can you please create an issue to disable this rule for tests?

Reply:
#25576